docs/source/install.rst

```diff
@@ -33,6 +33,21 @@ github using ``pip`` on a Python 3 environment: ::
 .. warning::
 
 	As of MRIQC 0.9.4, Python 2 is no longer supported.
+	
+matplotlib/Tcl warning::
+	MRIQC uses matplotlib to create graphics. By default matlpotlib 
+	uses interactive Tck interface, which requires graphic enveronment working. 
+	If such enveronment is not present (for ex. when running under tmux),
+	you may see an error: ::
+	
+		_tkinter.TclError: couldn't connect to display "localhost:10.0"
+	
+	To fix it you need uncomment "backend : Agg" line in configuration file 
+	of matplotlib. The location of configuration file can be retrieved
+	from python command prompt: ::
+	
+	> import matplotlib
+	> print(matplotlib.matplotlib_fname())
```

Comment:
```suggestion
	  >>> import matplotlib
	  >>> print(matplotlib.matplotlib_fname())

      Alternatively, you can issue the following shell command-line to edit this setting::
      
        $ sed -i 's/\(backend *: \).*$/\1Agg/g' $( python -c "import matplotlib; print(matplotlib.matplotlib_fname())" )

```

Reply:
Not sure that `sed` is needed for changing one value in one configuration file. 
Also a typo in line may lead to unpredictable results.

Reply:
Sure, that's why we are still providing both mechanisms for people to do this. In general, a typo on any command line could potentially lead to unpredictable results - I think we can trust users know that.

Reply:
Also, if you are using tmux or a headless terminal (e.g., on a cluster) that means you are probably familiar with some shell rudiments. And if you are not savvy on shell, I think copying and pasting this is much easier than running python and then getting trapped into `vim` or `emacs` or `nano`.

IMHO this is safe to suggest, and I'd be actually inclined to make the suggestion the default.

Reply:
You are the boss